src/en/getting-started-jaas.md

```diff
@@ -118,11 +120,40 @@ Enter the username and set the permissions for the new user.
 
 ![juju_jaas_share_screen](./media/juju_jaas_share_screen.png)
 
-## Next steps
+### Learn the CLI
 
 Most users find the flexibility and power of using JAAS and Juju from the CLI
 rewarding. See [Using JAAS from the command line][jaascli] for more information.
 
+### Destroy a model
+
+When you are done and want to destroy the model you created, click your name.
+
+![juju_jaas_top_panel_name](./media/jaas-username.png)
```

Comment:
I really don't like these heavily cropped images. they go against style and look really odd. I don't think it is needed in this case if we are clear - perhaps say "click on your username in the top left" or whtever